### PR TITLE
Add debug information on kebechet repos

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Related to #249 

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

LOGGER printing for debugging means

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
/lgtm

File: app.py [hunk 1]
```diff
@@ -189,6 +189,7 @@ def main(
         graph = GraphDatabase()
         graph.connect()
         kebechet_repos = graph.get_active_kebechet_github_installations_repos()
+         _LOGGER.debug("Got these active kebechet installation repos: %s", kebechet_repos)
```
Comment (xtuchyna) [inline app.py:192]:
```suggestion
        _LOGGER.debug("Got these active kebechet installation repos: %s", kebechet_repos)
```

**Pull request merged**